packages/core/src/transport/failedSendBeacon.ts

```diff
@@ -0,0 +1,32 @@
+import { isExperimentalFeatureEnabled } from '../domain/configuration'
+import { addMonitoringMessage, monitor } from '../domain/internalMonitoring'
+
+const FAILED_SEND_BEACON_FLUSH_INTERVAL = 2000
+
+export function startFlushFailedSendBeacons() {
+  if (!isExperimentalFeatureEnabled('lower-batch-size')) return
+
+  setInterval(monitor(flushFailedSendBeacon), FAILED_SEND_BEACON_FLUSH_INTERVAL)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏it could be interesting to add some explanations about your overall strategy, cases that you expect to handle or not to handle. Hard to guess why we need local storage or why we need a set interval.

Reply:
I've updated the PR description. Let me know what you think?

Reply:
FMU, the updated PR description provide more information about our goal with this PR, however it does not provide more reasoning about your strategy in this file, cf

> Hard to guess why we need local storage or why we need a set interval.

Reply:
I see. I added more precision in the `Implementation strategy` section of the description

Reply:
ok, btw It could be even more useful in the file directly for future reference.